api/base/serializers.py

```diff
@@ -7,12 +7,17 @@
 
 
 def _rapply(d, func, *args, **kwargs):
-    """Apply a function to all values in a dictionary, recursively."""
+    """Apply a function to all values in a dictionary, recursively. Handles lists and dicts currently,
+    as those are the only complex structures currently supported by DRF Serializer Fields."""
     if isinstance(d, collections.Mapping):
         return {
             key: _rapply(value, func, *args, **kwargs)
             for key, value in d.iteritems()
         }
+    if isinstance(d, list):
```

Comment:
I was going to, but I'm not actually using that, and I'm not sure that something like that would come up with the API. That being said, @samchrisinger has some uses for this function, so the plan is to move it over to website and add some tests. When tests are in, that would be a great time to add handling of the tuple.

Also, it gets a tuple, would you expect it to return as a list or as a tuple? If it's handled the way you mentioned above, tuples would be turned into lists. Which maybe fine? But that's the other reason I didn't want to just jump in and do that since I didn't have an immediate need for it with what I'm doing.